WalletWasabi.Fluent/Behaviors/TriggerStyleClassBehavior.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Data;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.Behaviors
+{
+	public class TriggerStyleClassBehavior : DisposingBehavior<Control>
```

Comment:
Fairly sure this isnt required.

@jmacato can probably weigh in.

I think this can be done simply with animating opacity.

Reply:
I checked and no, he does a complex trigger behaviors... this class seems necessary i think

Reply:
AFAIK there is no opportunity in Avalonia to trigger an animation on a specific event, so this is a workaround. If the `StyleTrigger` is set to `true`, it will add the animation style class.

Reply:
I removed this class, notifications will take care of the messages like "copied"